build.cake

```diff
@@ -318,13 +321,23 @@ Task("Publish-HomeBrew")
     Information("Hash for creating HomeBrew PullRequest: {0}", hash);
 });
 
+Task("Publish-GitHub-Release")
+    .Does(() =>
+{
+    
+    var packageFile = File("Cake-bin-v" + semVersion + ".zip");	
+    var packagePath = buildResultDir + packageFile;
+    
+    GitReleaseManagerAddAssets(userName, password, "cake-build", "cake", milestone, packagePath.ToString());
```

Comment:
@patriksvensson if the path that is passed into GRM doesn't exist, it will throw an exception, thus fail the build.